odxtools/comparam_subset.py

```diff
@@ -58,7 +60,7 @@ def _build_odxlinks(self) -> Dict[OdxLinkId, Any]:
 @dataclass()
 class ComplexComparam(BaseComparam):
     comparams: NamedItemList[BaseComparam]
-    allow_multiple_values: bool = False
+    allow_multiple_values: Optional[bool] = None
```

Comment:
xsd is clear about this one, if it's not specified in the xml, then it is automatically false

Reply:
yes, but in order to be able to write the files the same way they are read, it must be possible to distinguish "attribute not present" from "attribute is 'false'". that said, `if comparam_subset.allow_multiple_values` works as before because `if None` behaves as `if False`...